docs/scheduled-transactions/spec.md

```diff
@@ -1,294 +0,0 @@
-
-# Scheduled Transactions Spec 
-
-## Goals and Constraints  
-  
-- Allow transaction to be submitted without all the required signatures and provide functionality for each of the signers to submit their signatures independently after a transaction was created.  
-- Allow users to submit transactions to Hedera that will execute **once** all required signatures are acquired.  
-  
-**Constraints**  
-- Expiry time for the scheduled transactions defined by a global property with a time limit of `N` seconds (`ledger.schedule.txExpiryTimeSecs`). If `N` seconds pass from the creation of the scheduled transactions, and it haven't gathered all of the required signatures, Hedera nodes will clear them from memory/state.  
-- Throttles on every GRPC Operation that is defined for the new MVP Scheduled Transactions will be implemented.
-- Based on the 2 bullets above, there is implicit limit of pending scheduled transactions in the network enforced by `txExpiryTimeSecds` and the throttle on `ScheduleCreate` operation.
-  
-## GRPC Interface  
-  
-New [Schedule Service](https://github.com/hashgraph/hedera-services/blob/master/hapi-proto/src/main/proto/ScheduleService.proto) is added in the protobufs. It is responsible for both the **creation and management** of Scheduled transactions.  
-  
-New `HederaFunctionality`:  
-- `ScheduleCreate` - Creates new scheduled transaction.  
-- `ScheduleSign` - Signs an already created scheduled transaction.  
-- `ScheduleDelete` - Deletes an already created scheduled transaction. Must be signed by the specified `adminKey` .
-- `ScheduleGetInfo` - Returns information for an already created scheduled transaction.    
-  
-### ScheduleCreate  
-  
-Additional [`ScheduleCreateTransactionBody`](https://github.com/hashgraph/hedera-services/blob/master/hapi-proto/src/main/proto/ScheduleCreate.proto) is added in the protobufs. The message has the following format:  
-  
-```  
-message ScheduleCreateTransactionBody {  
-  bytes transactionBody // The transaction serialized into bytes that must be signed
-  Key adminKey // (optional) The Key which is able to delete the Scheduled Transaction (if tx is not already executed)
-  AccountID payerAccountID // (optional) The account which is going to pay for the execution of the Scheduled TX. If not populated, the scheduling account is charged for the execution of the scheduled TX
-  SignatureMap sigMap // (optional) Signatures that could be provided (similarly to how signatures are provided in ScheduleSign operation) on Scheduled Transaction creation
-  bytes memo // (optional) // Short publicly visible memo about the scheduled transaction. Max length 100 bytes. No guarantee of uniqueness.
-}  
-```  
-  
-**Optional properties**  
-  
-- `adminKey` is an optional field. If set, the specified `adminKey` is able to execute `ScheduleDelete` operation.  
```

Comment:
Should this doc be retained in docs folder for future reference?

Reply:
Was recommended to delete it to avoid possible confusion. 👍